docs/tutorials/HowToTestAModule.md

```diff
@@ -0,0 +1,45 @@
+# Setup a test environment for integration tests of a module
+
+In order to test a module in its lifecycle with its respective facade we offer the `Moryx.TestTools.IntegrationTest`.
+The package brings a `MoryxTestEnvironment<T>`.
+With this class you can first create mocks for all module facades your module dependents on using the static `CreateModuleMock<FacadeType>` method.
+Afterwards you can create the environment using an implementation of the `ServerModuleFacadeControllerBase`, an instance of the `ConfigBase` and the set of dependency mocks.
```

Comment:
i don't think `ServerModuleFacadeControllerBase` exist in MORYX 8. it is instead `ServerModuleBase<>`